lib/matplotlib/axes/_axes.py

```diff
@@ -94,18 +94,108 @@ def _plot_args_replacer(args, data):
 
 class Axes(_AxesBase):
     """
+    Build an `~.Axes` instance in `~.Figure` *fig* with
+    *rect=[left, bottom, width, height]* in `~.Figure` coordinates.
+
     The :class:`Axes` contains most of the figure elements:
     :class:`~matplotlib.axis.Axis`, :class:`~matplotlib.axis.Tick`,
     :class:`~matplotlib.lines.Line2D`, :class:`~matplotlib.text.Text`,
     :class:`~matplotlib.patches.Polygon`, etc., and sets the
     coordinate system.
 
+    Parameters
+    ----------
+
+    adjustable :  string
+        [ ``'box'`` | ``'datalim'`` | ``'box-forced'``]
+
+    alpha  :  float
+        the alpha transparency (can be None)
+
+    anchor  :  string
+        [ ``'C'``, ``'SW'``, ``'S'``, ``'SE'``, ``'E'``,
+        ``'NE'``, ``'N'``, ``'NW'``, ``'W'`` ]
+
+    aspect : string or float
+        ``'auto'``, ``'equal'`` or an aspect_ratio as a float.
+
+    autoscale_on  :  bool
+        whether or not to autoscale the *viewlim*
+
+    axisbelow  :  bool or string
+        draw the grids and ticks below  (*True*) or above (*False*) most
+        other artists, or below lines (``'line'``) but above patches.
+
+    cursor_props  :  tuple
+        a (*float*, *color*) tupple.
+
+    facecolor  :  color or string
+        None, or "inherit" from rcParams or a color spec
+
+    figure  : `~.Figure` instance
+
+    frameon  :  bool
+        draw the axes frame
+
+    label  :  string
+        the axes label
+
+    navigate  :  bool
+
+    navigate_mode  :  string or None
+        [ ``'PAN'`` | ``'ZOOM'`` | None ] the navigation toolbar button status
+
+    position  :  [left, bottom, width, height]
```

Comment:
This would be a good time to change the actual signature from     `def __init__(self, fig, rect, ...` to use 'position' instead of 'rect'; there is no support for a 'position' arg or kwarg as far as I can see.  Everything is in terms of the 'rect' positional argument.